src/Runner/PhpInterpreter.php

```diff
@@ -107,9 +107,16 @@ public function getVersion(): string
 	}
 
 
+	/** @deprecated  */
 	public function canMeasureCodeCoverage(): bool
 	{
-		return $this->info->canMeasureCodeCoverage;
+		return $this->info->codeCoverageEngine !== null;
+	}
+
+
+	public function getCodeCoverageEngine(): ?string
```

Comment:
What about getCodeCoverageEngines(): array ?

Reply:
And choose the one in CliTester? :+1:

Reply:
Yes.